packages/browser/jest.setup.js

```diff
@@ -2,7 +2,15 @@
 // https://github.com/jsdom/jsdom/issues/1612
 /* eslint-disable @silverhand/fp/no-mutation */
 /* eslint-disable unicorn/prefer-module */
```

Comment:
Should provide ending block to form a disable-enable-pair

Reply:
Will fix in the next PR.